### ISSUE TITLE
Check if FTP extension is loaded if ftp is used

### ISSUE THREAD

**Issue by JHGitty** [1up-lab/OneupFlysystemBundle#118]
I got this error:
```
 Notice: Use of undefined constant FTP_BINARY - assumed 'FTP_BINARY' 
```

Issue was that the php extension `ftp.so` was not loaded. I use PHP 7.0.13.
I think this bundle should include a check to only allow usage of the ftp feature if the "ftp.so" extension is loaded - otherwise it should throw a better understanding error message.

**Comment by piwi91** [comment]
Also an issue for me because we don't use an FTP connection, we don't include the FTP extension

**Comment by sonu27** [comment]
Issue for me too.

**Comment by amici** [comment]
An issue for anybody with PHP7 that didn't explicitly load the FTP module.

**Comment by bytehead** [comment]
Fixed by #128
